code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -130,14 +130,22 @@
 	. = ..()
 
 	// Deal with the source of this zombie corruption
-	//  Infection organ needs to be handled separately from mutant_organs
-	//  because it persists through species transitions
+	// Infection organ needs to be handled separately from mutant_organs
+	// because it persists through species transitions
 	var/obj/item/organ/internal/zombie_infection/infection
 	infection = C.getorganslot(ORGAN_SLOT_ZOMBIE)
 	if(!infection)
 		infection = new()
 		infection.Insert(C)
 
+	// since tongue tied requires empty hands to speak, zombies cannot talk since
+	// they are holding claws (so we need to ditch the tongue if present)
+	var/obj/item/organ/internal/tongue/tied/bad_tongue = C.getorganslot(ORGAN_SLOT_TONGUE)
+	if(bad_tongue)
+		var/obj/item/organ/internal/tongue/zombie/good_tongue = new()
+		bad_tongue.Remove(C)
```

Comment:
qdel the old tongue so it doesnt just sit in nullspace

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/69840/commits/2ebe7fd9315bbf928a722dad8502e2742a15e9fd

Thanks for pointing it out since I thought removed organs just naturally qdel.